drivers/scheduler/k8s/k8s.go

```diff
@@ -7517,3 +7528,25 @@ func ClusterVersion() (string, error) {
 	}
 	return strings.TrimLeft(ver.String(), "v"), nil
 }
+
+func createClonedStorageClassIfRequired(originalStorageClass *storageapi.StorageClass) (string, error) {
+	clonedSCName := originalStorageClass.Name + "-clone"
+	clonedSCobj, err := k8sStorage.GetStorageClass(clonedSCName)
```

Comment:
A `Get` method returning an error does not necessarily mean the object does not exist. The error should be checked for network issues.

Reply:
made changes to explicitly check for not found errors.